### PR TITLE
fix!: `/parsed_to_cn_var` should accept number, definite range, and indefinite range

### PR DESCRIPTION
**Pull request by korikuzma**
addresses #443 for metaschema . This work is needed for the manuscript analysis

- Merges `parsed_to_cn_var` and `parsed_to_cx_var work` into `parsed_to_copy_number`
- ac/chr/assembly check is now done in the Pydantic model

### CODE REVIEW

File: variation/main.py [hunk 1]
```diff
@@ -541,13 +541,21 @@ async def hgvs_to_copy_number_change(
...
+end_pos_type = "Type of the end value in VRS Sequence Location"
+assembly_descr = ("Assembly. If `accession` is set, will ignore `assembly` and `chr`. "
+                  "If `accession` not set, must provide both `assembly` and `chr`.")
 chr_descr = "Chromosome. Must set when `assembly` is set."
-accession_descr = "Accession. If `accession` is set, will ignore `assembly` and "\
-                  "`chr`. If `accession` not set, must provide both `assembly` and `chr`."  # noqa: E501
-start_descr = "Start position as residue coordinate"
-end_descr = "End position as residue coordinate"
+accession_descr = ("Accession. If `accession` is set, will ignore `assembly` and `chr`."
+                   " If `accession` not set, must provide both `assembly` and `chr`.")
```
Comment (jsstevenson) [inline variation/main.py:558]: @korikuzma Maybe I'm reading this wrong -- but what happens if I set both `assembly` and `accession`?

Comment (korikuzma) [reply variation/main.py:558]: If `accession` is set, it will use that. Should I fix the wording?

Comment (korikuzma) [reply variation/main.py:558]: Maybe I should put that this is just for genomic atm.

Comment (jsstevenson) [reply variation/main.py:558]: Oh, I'm sorry, I was misreading. I might suggest rephrasing to something like this for clarity

```
assembly_descr = ("Assembly. Ignored, along with `chr`, if `accession` is set. "
                  "If `accession` not set, must provide both `assembly` and `chr`.")
```

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**